source/libs/api.js

```diff
@@ -1,18 +1,40 @@
 /*
+These will throw `RefinedGitHubAPIError` if something goes wrong or if it's a 404.
+Probably don't catch them so they will appear in the console
+next to the name of the feature that caused them.
+
 Usage:
 
 import * as api from '../libs/api';
-
 const user  = await api.v3(`users/${username}`);
-const {data} = await api.v4(graphQuery);
+const data = await api.v4('{user(login: "user") {name}}');
+
+Returns:
+a Promise that resolves into an object.
+
+If the response body is empty, you'll receive an object like {status: 200}
+
+The second argument is an options object, it lets you defined accepted error codes, like:
 
+{
+	accept404: true
+	accept500: true
```

Comment:
I would prefer one option for this instead of a separate option for each code.

Maybe `{acceptedErrorStatusCodes: [404]}`?

Reply:
It’s not really needed at this point, I just felt that it could be shortened by extending its functionality. If we have to make it longer we might as well go back to a single `accept404`. 😅 what do you think?

Reply:
If we don't need to support this for any other codes (?), I would just go back to `accept404`.